cypress_shared/pages/page.ts

```diff
@@ -158,4 +158,10 @@ export default abstract class Page {
       cy.readFile(downloadedFilename, 'binary', { timeout: 300 })
     })
   }
+
+  shouldShowCyaTitle(taskName: string, taskTitle: string) {
```

Comment:
Can we call this `shouldShowCheckYourAnswersTitle`? It's a bit more verbose, I know, but the `cya` naming confused me a bit to start with

Reply:
Yeah fair enough!